lib/ransack_ui/ransack_overrides/helpers/form_builder.rb

```diff
@@ -143,7 +143,7 @@ def attribute_collection_for_base(base, attributes=nil)
           if c == 'id'
             foreign_klass = object.context.traverse(base).model_name
             # Check that model can autocomplete. If not, skip this id column.
-            next nil unless foreign_klass.constantize._ransack_autocompletes_through.present?
+            next nil unless foreign_klass.to_s.constantize._ransack_autocompletes_through.present?
```

Comment:
what is this even doing... 
`.model_name` and then `.constantize`? Why not just use the `foreign_klass` directly?